apps/prairielearn/src/models/questions.sql

```diff
@@ -0,0 +1,44 @@
+-- BLOCK select_questions_for_course
+WITH
+  issue_count AS (
+    SELECT
+      i.question_id,
+      count(*) AS open_issue_count
+    FROM
+      issues AS i
+    WHERE
+      i.course_id = $course_id
+      AND i.course_caused
+      AND i.open
+    GROUP BY
+      i.question_id
+  )
+SELECT
+  q.id,
+  q.qid,
+  q.title,
+  q.sync_errors,
+  q.sync_warnings,
+  q.grading_method,
+  q.external_grading_image,
+  case
+    when q.type = 'Freeform' then 'v3'
+    else 'v2 (' || q.type || ')'
+  end AS display_type,
+  coalesce(issue_count.open_issue_count, 0) AS open_issue_count,
+  row_to_json(top) AS topic,
+  tags_for_question (q.id) AS tags,
+  assessments_format_for_question (q.id, NULL) AS assessments
+FROM
```

Comment:
@mwest1066 @nwalters512 this is the only place in the whole codebase that the `assessments_format_for_question` sproc is called, do you want me to just inline it here so we can get rid of it? 
or should we just leave it for now?

Reply:
There are a few others:
https://github.com/PrairieLearn/PrairieLearn/blob/a475b682d9664e9ff4b34ddc36ea6564def4631f/apps/prairielearn/src/middlewares/selectAndAuthzInstructorQuestion.sql#L87
https://github.com/PrairieLearn/PrairieLearn/blob/a475b682d9664e9ff4b34ddc36ea6564def4631f/apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.sql#L83

Reply:
oh sorry, I think my editor was restricted to a certain folder when I searched it. 

What is the preferred thing to do here then? The query is quite large to have inlined multiple places.

Reply:
Keeping this sproc as-is is fine for now.

In the future, we should consider inlining it in the questions query here. I think we could make the argument that showing the "other assessments" on the instructor assessments questions page isn't super valuable? That just leaves `selectAndAuth...`, and it might not be too terrible to include it inline there too.

Reply:
sounds good